pylint_google_style_guide_imports_enforcing/test_checker.py

```diff
@@ -0,0 +1,34 @@
+import astroid
+import pylint.testutils
+
+from pylint_google_style_guide_imports_enforcing import AlphabeticallySortedImports
+
+
+class TestAlphabeticallySortedImports(pylint.testutils.CheckerTestCase):
+    CHECKER_CLASS = AlphabeticallySortedImports
+
+    def test_import_exceptions_with_proper_imports(self):
```

Comment:
That test name is a bit tricky - using words like correct, valid, proper etc is a slippery slope because what was correct/valid/proper today may not be valid/correct/proper tomorrow 🤫 

How about `test_doesnt_report_direct_imports_from_excluded_modules` ?

Reply:
Fixed :+1: